babel.cfg

```diff
@@ -1,3 +1,8 @@
 [python: **.py]
-[jinja2: **/templates/**.jinja2]
+[jinja2: **/templates/**.xml.jinja2]
 extensions=jinja2.ext.autoescape,jinja2.ext.with_,jinja2.ext.do
+[jinja2: **/templates/**.html.jinja2]
+extensions=jinja2.ext.autoescape,jinja2.ext.with_,jinja2.ext.do,webassets.ext.jinja2.AssetsExtension
+[javascript: **/templates/js/**.js.jinja2]
```

Comment:
The `js/` folder is unnecessary.

Reply:
Made is explicit because it's where all the JS templates are.